tests/udf/test_simple_udf.py

```diff
@@ -53,3 +55,26 @@ def my_frame_fn(frame, pixelsum):
     assert 'pixelsum' in res
     print(data.shape, res['pixelsum'].data.shape)
     assert np.allclose(res['pixelsum'].data, np.sum(data, axis=(1, 2)))
+
+
+def test_minibatch(lt_ctx):
+    data = _mk_random(size=(16, 16, 16, 16), dtype="float32")
+    dataset = MemoryDataSet(data=data, tileshape=(1, 1, 16, 16),
+                            partition_shape=(4, 4, 16, 16), sig_dims=2)
+
+    res = lt_ctx.run_udf(
```

Comment:
You should also have a test case that calls your top-level interface function (that is currently called `run_analysis`, see comment above). You can keep a test with `run_udf` if you want to check any intermediate results that may not be returned from the `run_analysis` function.

Reply:
For now, there is not much of a difference between run_udf and run_stddev so I just removed the test on run_udf